llvm/lib/Target/SPIRV/SPIRVInstructionSelector.cpp

```diff
@@ -2653,6 +2653,22 @@ bool SPIRVInstructionSelector::selectIntrinsic(Register ResVReg,
         .addUse(GR.getSPIRVTypeID(ResType))
         .addUse(GR.getOrCreateConstInt(3, I, IntTy, TII));
   }
+  case Intrinsic::spv_wave_read_lane_at: {
+    assert(I.getNumOperands() == 4);
```

Comment:
Put this block in a select function. It makes it easier to see all the various case statements when scrolling through the intrinsic selector.